dist/index.html

```diff
@@ -9,24 +9,24 @@
 <body>
   <header>
     <h1>The Overlook Hotel</h1>
-    <button type="button" name="button" class="add-new-bookings-button" id="addNewBookingsButton">Add New Booking</button>
+    <button type="button" name="button" class="add-new-bookings-button medium-button hide" id="addNewBookingsButton">New Booking</button>
   </header>
   <main>
     <section class="login-view" id="loginView">
       <form class="login-container" id="loginContainer">
-        <h2>Login To Your Account</h2>
-        <label for="userName">UserName</label>
-        <input type="text" name="" value="" id="userNameInput" required>
+        <h2 class="login-title">The Overlook Hotel</h2>
+        <label for="userName">Username</label>
+        <input type="text" name="" value="customer25" id="userNameInput" required>
```

Comment:
Such a good idea hardcoding those values during buildout

Reply:
Yep makes it easy to deal with